src/components/DropDown/DropDown.js

```diff
@@ -185,17 +188,19 @@ const DropDown = React.memo(function DropDown({
 DropDown.propTypes = {
   header: PropTypes.node,
   items: PropTypes.arrayOf(PropTypes.node).isRequired,
-  label: PropTypes.string,
+  placeholder: PropTypes.string,
+  renderLabel: PropTypes.string,
```

Comment:
We usually use the `render` prefix for render functions, maybe we could call it and pass the selected index?

```suggestion
  renderLabel: PropTypes.func,
```

Reply:
And as a default prop value: `index => items[index]`

Reply:
Yes good idea, having a default value can also serve as an example when looking at the props in the code. :+1: